core/src/test/java/de/evoila/cf/broker/controller/utils/DashboardUtilsTest.java

```diff
@@ -0,0 +1,767 @@
+package de.evoila.cf.broker.controller.utils;
```

Comment:
If you found a way to make the test class in https://github.com/evoila/osb-core/pull/57 smaller or split it, then this test class would be a perfect candidate for it as well. 😄